### DISCUSSION TITLE
Validating the range picker to ensure endDate is after the startDate

### DISCUSSION THREAD

**Discussion by anchovy** [easepick/easepick#235] [Q&A]
Hi - we've using the RangePlugin and TimePulgin and we need to validate that the endDate is after the startDate.

We're-pre populating the start/end date (if it's being edited) but I can't work out how to check the new selected dates validate.

in the on-change function it's only ever giving me the initial start/end date and not the 'new' dates.

```
        picker.on("change", (e) => {
          console.log(picker.getStartDate());
         console.log( picker.getEndDate());
   });
   ```

Any ideas or tips on how to implement this on date/time select or on clicking apply would be much appreciated!

**Comment by cavasinf** [comment]
How did you manage to get into this case?
![image](https://github.com/easepick/easepick/assets/25293190/522d38ef-61b8-453a-9279-45f68bdf03bb)

You have two different inputs, one for start and one for end?
Have you tried `elementEnd`?
![image](https://github.com/easepick/easepick/assets/25293190/da72957e-f2fa-4818-b2d6-d2ddf5f5738b)
